tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
     "compilerOptions": {
         "outDir": "./dist",
-        "target": "es5",
+        "target": "ES6",
```

Comment:
You've upgraded your ES version here, is that intentional?

Reply:
I've downgraded this to ES5, I had upgraded it during the initial debugging attempts in case that was where the syntax error was coming from as some people had said that was where their issues were coming from. Everything does seem to be working fine when I set it back to ES5 though, so I've reverted it.